src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -177,6 +180,11 @@ void GraphingCalculator::OnTracePointChanged(Windows::Foundation::Point newPoint
     PositionGraphPopup();
 }
 
+void CalculatorApp::GraphingCalculator::OnPointerPointChanged(Windows::Foundation::Point newPoint)
+{
+    TracePointer->Margin = Thickness(newPoint.X, newPoint.Y, 0, 0);
```

Comment:
>TracePointer->Margin = Thickness(newPoint.X, newPoint.Y, 0, 0); [](start = 4, length = 63)

Moving the pointer by setting margins on it...  Not sure this is a good way to go about it, but didn't see any absolute positioning apis...

Reply:
Maybe using a [canvas](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.canvas) could help, but the margins seem to work fine.